srcpkgs/exult/template

```diff
@@ -0,0 +1,50 @@
+# Template file for 'exult'
+pkgname=exult
+version=1.12.0
+revision=1
+archs="x86_64*" # Test suite failure on i686
```

Comment:
Instead of restricting the architectures mark a specific architecture as broken if required or skip tests for that architectures. `archs` is to limit for architectures it actually doesn't support.

```
case "${XBPS_TARGET_MACHINE}" in
i686) broken="test suite failure" ;;
esac